plugins/sfIsdfPlugin/modules/sfIsdfPlugin/templates/indexSuccess.php

```diff
@@ -20,6 +17,17 @@
     </div>
   <?php } ?>
 
+  <section class="breadcrumb">
+    <ul>
+    <li><?php echo link_to(esc_specialchars(sfConfig::get('app_ui_label_function')), ['module' => 'function', 'action' => 'browse']); ?></li>
+      <li><span><?php echo render_title($resource); ?></span></li>
+    </ul>
+  </section>
+
+<?php end_slot(); ?>
+
+<?php slot('before-content'); ?>
+
```

Comment:
Apart from helping with the implementation I think this is a good change, but I'd like to check with @fiver-watson ...

Basically, it removes the "View ISDF function" from the function's index page heading and adds a breadcrumb bellow the function title. This breadcrumb uses the UI label for functions and links to the function's browse page.

Reply:
Looks good to me - thanks for noticing and addressing this, @mInnes-archives ! The functions module has been a bit neglected, as it is not frequently used in most North American archival practices (or some European and others - it's most often used in the Australian series system). However, I agree that this is a good change, that brings the page layout more in line with how other entities are presented. Thanks again!